spec/acceptance/svn_paths_spec.rb

```diff
@@ -5,7 +5,6 @@
 describe 'subversion :includes tests on SVN version >= 1.7', unless: ( # rubocop:disable RSpec/MultipleDescribes : The
     # test's on this page must be kept seperate as they are for different operating systems.
     (fact('osfamily') == 'RedHat' && fact('operatingsystemmajrelease') =~ %r{^(5|6)$}) ||
-    (fact('osfamily') == 'Debian' && fact('operatingsystemmajrelease') =~ %r{^(6|7|10\.04|12\.04)$}) ||
```

Comment:
You've removed all versions here instead of just 6 & 7

Reply:
Ubuntu 12 and 10 are unsupported.

Reply:
Woops, yep. You're right.